src/common/node/childProcess.ts

```diff
@@ -16,7 +16,8 @@ export interface ISpawnResult {
     stdin: NodeJS.WritableStream;
     stdout: NodeJS.ReadableStream;
     stderr: NodeJS.ReadableStream;
-    outcome: Q.Promise<void>;
+    startup: Q.Promise<void>; // The app started succesfully
```

Comment:
I'm not sure. The problem is that we are kinda using promises as event. waitForStart and waitForFinish are great names for events, but they are kind of weird for promises. Promises are more about getting a result, so for promises it might make more sense to call it startupResult or processResult or outcome. So I'm not sure if we should go with event kind of names, or promises kind of names.
If you are convinced these names make more sense, I'll change them!